master/static/migrations/20230208142448_add-update-modified-time-user-table.tx.up.sql

```diff
@@ -0,0 +1,16 @@
+CREATE OR REPLACE FUNCTION public.set_modified_time ()
+    RETURNS TRIGGER
+    AS $$
+BEGIN
+    IF (TG_OP = 'UPDATE') THEN
+        NEW.modified_at := now();
```

Comment:
question: not a postgresql expert, but would `clock_timestamp()` be more accurate than `now()`?
[Relevant read](https://dba.stackexchange.com/questions/250836/postgres-trigger-always-uses-the-same-value-for-now)

Reply:
Looks like both are the same things according to https://dba.stackexchange.com/a/63549 (i just read the first line)

Reply:
this article didnt talk about `clock_timestamp`. nvm

Reply:
so according to the article, it says `now()` and `CURRENT_TIMESTAMP` are the same things.
`users` table looks like `modified_at timestamp without time zone DEFAULT CURRENT_TIMESTAMP`.
I think its good to have compatibility with `CURRENT_TIMESTAMP`, but im not postgres expert either, so idk which one is suitable in this case.

Reply:
ended up using `now()` after [discussion](https://determined-ai.slack.com/archives/CSG3W08JY/p1675902804760329)

Reply:
cool, the experts have spoken